Terasology/CombatSystem
# Ammo slot issue fixed, and name change of method used

## Summary

### Result

* When opening a save file of a Light And Shadow game in which the last selected item was a bow, the Quiver inventory slot is visible.

### Original Issue 

* Under the above circumstances, the quiver inventory slot was not visible. Instead, it became visible when you scrolled twice, deselecting and selecting again the bow.

## Contents

### Changes

* The method which modifies the slot's visibility was only received through scrolling, thus adding `CharacterComponent.class` within the `@ReceiveEvent`. I guessed that this method has to run within the start-up of the save file, as it represented information for the user's character.
* Changed the method's name from `onLauncherSelected` to `onAmmoBearingItemSelected`, as Cervator suggested. This name is more accurate, according to the method's actions.

## How to test

* Create a LightAndShadow game
* Check 1: see if ammo inventory slot is visible

Continue testing by:

* Closing terasology
* Opening the save file you created
* Check 2: see if ammo inventory slot is visible

I mention "Check 1", because even if he issue did not occur when -creating- a game, it did have an issue if the order of `haracterComponent.class` and `CharacterHeldItemComponent.class` within:
```Java 
@ReceiveEvent(components = {CharacterComponent.class, CharacterHeldItemComponent.class}) 
```
then the ammo inventory slot didn't appear at all, even with scrolling.

## Screenshot

I marked with red:

* At the left: the bow that you should have preselected at the inventory
* At the right: the ammo slot that you should see if it is visible when the bow is selected

<img width="889" alt="ammo indication" src="https://user-images.githubusercontent.com/48293545/86487564-146a1880-bd67-11ea-8456-57dea2e394cd.png">
